src/api/pipeline/client.ml

```diff
@@ -0,0 +1,84 @@
+open Capnp_rpc_lwt
+
+module State = struct
+  open Raw.Reader.JobInfo.State
+
+  type t = unnamed_union_t
```

Comment:
I don't understand where this type comes from, is it a placeholder?

Reply:
I'm not sure either. I've replaced it with an explicit sum type.